tests/test_pyn5.py

```diff
@@ -33,8 +33,10 @@ def tearDown(self):
 
         def test_read_write_valid(self):
             self.n5.write_block([0, 0, 0], self.valid_block)
-            self.assertEqual(
-                self.n5.read_ndarray([0, 0, 0], self.block_size), self.valid_block
+            a = self.n5.read_ndarray([0, 0, 0], self.block_size)
```

Comment:
it looks like a is assigned but never used

Reply:
Accidental remnant of some debugging. Force pushed and will see if that passes.